### PR TITLE
Nisp 2343

### PR DESCRIPTION
**Pull request by nehakannaujia**
Exclusion unit tests

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -15,3 +15,5 @@ resolvers += Resolver.url("scoverage-bintray", url("https://dl.bintray.com/sksam
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.3.0")
 
 addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "0.6.0")
+
+addSbtPlugin("com.typesafe.sbt" % "sbt-digest" % "1.0.0")
```
Comment (dspork) [inline project/plugins.sbt]: Is this required?

File: test/uk/gov/hmrc/nisp/views/HtmlSpec.scala [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+import uk.gov.hmrc.play.http.SessionKeys
+import uk.gov.hmrc.play.partials.CachedStaticHtmlPartialRetriever
+import uk.gov.hmrc.play.test.UnitSpec
+import uk.gov.hmrc.time.DateTimeUtils.now
+import play.twirl.api.Html
+import org.apache.commons.lang3.StringEscapeUtils
+import uk.gov.hmrc.nisp.controllers.auth.NispUser
+import uk.gov.hmrc.play.frontend.auth.connectors.domain.{Accounts, ConfidenceLevel, CredentialStrength, PayeAccount}
+
+trait HtmlSpec extends UnitSpec  {
```
Comment (dspork) [inline test/uk/gov/hmrc/nisp/views/HtmlSpec.scala]: Can we reformat this document?

In IntelliJ you can use `Code -> Refomat Code` or `Ctrl` + `Alt` + `L`

File: test/uk/gov/hmrc/nisp/views/ExclusionViewSpec.scala [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    override val sessionCache: SessionCache = MockSessionCache
+    override val metricsService: MetricsService = MockMetricsService
+    override implicit val cachedStaticHtmlPartialRetriever: CachedStaticHtmlPartialRetriever = MockCachedStaticHtmlPartialRetriever
+  }
+
+  "Exclusion Dead template" should {
+    "render page with heading  You are unable to use this service " in {
+      var sResult = html.excluded_dead(List(Exclusion.Dead) , Some(65)) ;
+      val htmlAccountDoc = asDocument(contentAsString(sResult));
+      assertElementContainsText(htmlAccountDoc ,"article.content__body>h1" , Messages("nisp.excluded.title"))
```
Comment (dspork) [inline test/uk/gov/hmrc/nisp/views/ExclusionViewSpec.scala]: Maybe we could use `assertEqualsMessage` and pass in the key? As this also tests the key has a value.

File: project/plugins.sbt (hunk 1 above)
Comment (nehakannaujia) [inline project/plugins.sbt]: No not really, i added it for cache task. I will remove it.

File: test/uk/gov/hmrc/nisp/views/HtmlSpec.scala (hunk 2 above)
Comment (nehakannaujia) [inline test/uk/gov/hmrc/nisp/views/HtmlSpec.scala]: Sure

File: test/uk/gov/hmrc/nisp/views/ExclusionViewSpec.scala (hunk 3 above)
Comment (nehakannaujia) [inline test/uk/gov/hmrc/nisp/views/ExclusionViewSpec.scala]: yeah its good option,its checking Message key also. I will do the changes.

**Pull request merged**